### ISSUE TITLE
Can we have an example of what you can *do* with a colour stack?

### ISSUE THREAD

**Issue by chipcullen** [at-import/toolkit#28]
I just have a question on actual usage for color stacks - I love the concept, but am a little lost on actually using the results. As far as I can tell, per the code in the readme, you can set a variable with the colour stack function:

```
$red-blue: colour-stack(red, blue, 25%, 50%, 75%, 100%);
```

Sooooo - what can you now do with your $red-blue variable? Since the processed SCSS results in 

```
red, #bf003f, #7f007f, #3f00bf, blue;
```

Which, is great. But you really can't pass that as a value into your implementation, unless there is a way to use SASS variables with multiple values that I'm not aware of.  

I guess I would just love to see a little more explanation on usage. Or, put more bluntly: You invoke colour-stack - now what?

Thank you for a great set of tools! Like I say, I love this idea, and want to use it - I'm just not quite grasping how to use the results.

**Comment by Snugug** [comment]
The `nth` function is how you get a color out. So, if you wanted a color that was right between red and blue, you'd use `nth($red-blue, 3)`. If you wanted red you'd use `nth($red-blue, 1)`. If you wanted a red-blue mix that was more blue than red, you'd use `nth($red-blue, 4)`.

While red/blue may not be the perfect real-world example, I use tint/shade stacks (a form of a color stack) all the time to provide me with color variations that I can easily pick out. Recently, however, I worked on a project with 18 shades of grey, so I used color stacks to build three sets of grey, dark, mid, and light, for me to be able to pick off the colors I wanted.
